spec/acceptance/init_spec.rb

```diff
@@ -39,7 +39,7 @@
       result = run_bolt_task('service', 'action' => 'status', 'name' => package_to_use)
       expect(result.exit_code).to eq(0)
       expect(result['result']).to include('status' => %r{running|Started})
-      expect(result['result']).to include('enabled' => %r{true|Manual|Automatic})
+      expect(result['result']).to include('enabled' => %r{true|manual|automatic})
```

Comment:
```suggestion
      expect(result['result']).to include('enabled' => %r{true|manual|automatic}i)
```
If case is a problem I think its safe to ignore in the regex.

Reply:
Thanks @donoghuc , i've updated the code. Close this PR and opened another [one](https://github.com/puppetlabs/puppetlabs-service/pull/134) using my fork.